examples/EmulatorDemo.py

```diff
@@ -0,0 +1,90 @@
+"""
+ This Demo shows how to emulate the recorded data with the SensorEmulator to create a live plotter.
+ The live plotter code is almost directly copied from the LivePlotterDemo.py but the SensorManager is swapped for the sensor emulator
+
+ If you didn't already record a sensor with the RecorderDemo.py yet, do that first because that data is needed for this demo.
+"""
+
+import time
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+from matplotlib import style
+
+# Import the CreaTeBME package after installing it with: `pip install CreaTeBME`
+from CreaTeBME import SensorEmulator
+
+# Constants:
+sample_rate = 100
+axis_names = ["Accelerometer x", "Accelerometer y", "Accelerometer z", "Gyroscope x", "Gyroscope y", "Gyroscope z"]
+axis_colors = ["red", "green", "blue"]
+
+# Style the graph (optional)
+style.use('seaborn')
```

Comment:
`MatplotlibDeprecationWarning: The seaborn styles shipped by Matplotlib are deprecated since 3.6, as they no longer correspond to the styles shipped by seaborn. However, they will remain available as 'seaborn-v0_8-<style>'. Alternatively, directly use the seaborn API instead.`